### PR TITLE
Closes #254: implement New Address screen

### LINKED ISSUE #254: Implement New Address Screen
https://projects.invisionapp.com/d/main/default/#/console/13241316/282749289/preview

### PR DESCRIPTION
**Pull request by valentinewallace**
Very similar to DepositView

### CODE REVIEW

File: src/view/new-address.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  },
+});
+
+const NewAddressView = ({ store, invoice }) => (
+  <SplitBackground image="purple-gradient-bg" bottom={color.blackDark}>
+    <View style={styles.qrWrapper}>
+      <QRCode size={130}>{store.walletAddressUri}</QRCode>
+    </View>
+    <MainContent style={styles.content}>
+      <CopySection />
```
Comment (tanx) [inline src/view/new-address.js]: Wrapped the copy text in a CopiedSection like in the SeedView.

File: src/view/new-address.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    alignItems: 'center',
+  },
+  copyBtn: {
+    backgroundColor: color.blackDark,
+    marginBottom: 40,
+  },
+  copied: {
+    bottom: 180,
+  },
+});
```
Comment (tanx) [inline src/view/new-address.js]: Played around with the margins to make it more aligned with the design.

File: src/view/seed.js [hunk 3]
```diff
@@ -38,10 +38,10 @@ const copyStyles = StyleSheet.create({
   },
   title: {
     textAlign: 'center',
-    marginBottom: 15,
   },
   copyTxt: {
     textAlign: 'center',
+    marginTop: 15,
```
Comment (tanx) [inline src/view/seed.js]: This was just something I saw in the SeedView. We should always add `topMargin` to the following element. I think I added this the wrong way and you did it right here.

File: stories/screen.js [hunk 4]
```diff
@@ -66,6 +67,7 @@ storiesOf('Screens', module)
   .add('Loader - First Time', () => <Loader />)
   .add('Seed', () => <Seed store={store} />)
   .add('Seed Success', () => <SeedSuccess />)
+  .add('New Address', () => <NewAddress store={store} invoice={invoice} />)
```
Comment (tanx) [inline stories/screen.js]: This is just nitpicking, but I reordered the screens so they are in line with the user journey. This way it's easier to follow in storybook.

File: src/view/seed.js (hunk 3 above)
Comment (valentinewallace) [reply src/view/seed.js]: ah yeah good to be consistent about that

**Pull request merged**